docs/override-docker-services.md

```diff
@@ -0,0 +1,59 @@
+## Overriding configs in a custom Ozone distribution
+
+- Create a custom Ozone distribution following [this](https://docs.ozone-his.com/create-distro/) guide.
+- Add the custom docker compose override configurations and make sure they're copied to the right directory.
+- Add the build step in the `pom.xml`file for the custom configs:
+
+```
+    <execution>
+      <!-- Copy over the Docker Compose override file -->
+        <id>Copy docker-compose-override.yml</id>
+        <phase>process-resources</phase>
+        <goals>
+          <goal>copy-resources</goal>
+        </goals>
+        <configuration>
+          <outputDirectory>
+            ${project.build.directory}/${project.artifactId}-${project.version}/run/docker/
+          </outputDirectory>
+          <overwrite>true</overwrite>
+          <resources>
+            <resource>
+              <directory>${project.basedir}/scripts</directory>
+              <includes>
+                <include>docker-compose-override.yml</include>
+              </includes>
+              <filtering>true</filtering>
+            </resource>
+          </resources>
+        </configuration>
+      </execution>
+```
+
+- Build the project by running `./scripts/mvnw clean package`
+- Start the Ozone services by running: 
+```
+source target/go-to-scripts-dir.sh
+./start-demo.sh
+```
+
+## Disabling components in Ozone distribution
+
+Ozone distribution comes with OpenMRS 3, Odoo, SENAITE and Superset as the default components. If you want to disable any of them, here are the steps to follow:
+
+ - Start Ozone using [quick start](https://docs.ozone-his.com/) guide.
+
+ - Stop the service by running `destroy-demo.sh`. In the scripts directory `target/ozone/run/docker/scripts/`, remove the manifest in `docker-compose-files.txt` for the component you want to disable. For example if you want to disable SENAITE component, remove `docker-compose-senaite.yml`.
+ - Restart Ozone by running `./start-demo.sh`.
```

Comment:
About enabling/disabling components, let's make sure we feed from @corneliouzbett's input here if we are not yet stating those elements →  https://openmrs.slack.com/archives/C02PYQD5D0A/p1718883883252649?thread_ts=1718873462.534859&cid=C02PYQD5D0A